lib/sisimai/bite/email/imailserver.rb

```diff
@@ -0,0 +1,181 @@
+module Sisimai::Bite::Email
+  module IMailServer
+    # Sisimai::Bite::Email::IMailServer parses a bounce email which created by
+    # Ipswitch IMail Server.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite::Email/IMailServer.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'x-mailer' => %r/\A[<]SMTP32 v[\d.]+[>][ ]*\z/,
+        :'subject'  => %r/\AUndeliverable Mail[ ]*\z/,
+      }
+      Re1 = {
+        :begin  => %r/\A\z/,    # Blank line
+        :error  => %r/Body of message generated response:/,
+        :rfc822 => %r/\AOriginal message follows[.]\z/,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReSMTP = {
+        conn: %r{(?:
+             SMTP[ ]connection[ ]failed,
+            |Unexpected[ ]connection[ ]response[ ]from[ ]server:
+            )
+        },
+        ehlo: %r|Unexpected response to EHLO/HELO:|,
+        mail: %r|Server response to MAIL FROM:|,
+        rcpt: %r|Additional RCPT TO generated following response:|,
+        data: %r|DATA command generated response:|,
+      }
+      ReFailure = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->